src/paulscode/android/mupen64plusae/MenuListView.java

```diff
@@ -180,10 +211,26 @@ public View getChildView( int groupPosition, final int childPosition, boolean is
                 TextView text1 = (TextView) view.findViewById( R.id.text1 );
                 TextView text2 = (TextView) view.findViewById( R.id.text2 );
                 ImageView icon = (ImageView) view.findViewById( R.id.icon );
+                ImageView indicator = (ImageView) view.findViewById( R.id.indicator );
                 
                 text1.setText( item.getTitle() );
                 text2.setVisibility( View.GONE );
                 icon.setImageDrawable( item.getIcon() );
+                
+                view.setBackgroundColor( 0x0 );
+                
+                // Indent child views by 15 points
+                DisplayMetrics metrics = new DisplayMetrics();
+                ((Activity) mListView.getContext()).getWindowManager().getDefaultDisplay().getMetrics(metrics);
+                
+                view.setPadding( (int) ( 15 * metrics.density ), view.getPaddingTop(), view.getPaddingRight(), view.getPaddingBottom() );
+                
+                if ( !item.isCheckable() )
```

Comment:
Added the ability for child rows to have checkboxes using the ImageView on the right side.